### ISSUE TITLE
Support native VS Code citation picker

### ISSUE THREAD

**Issue by jooyoungseo** [mblode/vscode-zotero#31]
Currently, this extension only uses Zotero's quick citation picker. However, it would be even nicer if users could switch to native VS Code citation picker, which could be much more accessible and seamless. FYI, the native citation picker works better for screen readers and other assistive technologies.

For example, [tecosaur/LaTeX-Utilities#29](https://github.com/tecosaur/LaTeX-Utilities/pull/29) allows this feature:

``` json
{
    "latex-utilities.zotero.citeMethod": "vscode"
}
```

However, that extension currently do not support `Pandoc` style citation.

**Comment by mblode** [comment]
Agreed this would be a nice accessibility win. Building a native VS Code QuickPick that talks to Better BibTeX (and supports Pandoc-style output) is a sizeable feature, so I'm keeping this open and labelling it `feature`. PRs very welcome.
